<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Main</class>
 <widget class="QWidget" name="Form_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 700 30pt &quot;霞鹜文楷等宽&quot;;
}

.QWidget{
	border:1px solid black;
	border-radius: 20px;
}

.QGroupBox{
	border:1px solid black;
	border-radius: 20px;
}

QLabel{
	font:30pt;
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 250, 216);
}

QPushButton{
	font: 24pt;
	border:1px solid black;
	border-radius:25px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:1, stop:0 rgba(255, 250, 233, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	font: 25pt;
	background-color: qlineargradient(spread:pad, x1:0.66, y1:0.66, x2:0, y2:0, stop:0 rgba(250, 249, 230, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	font: 25pt;
	border:1px solid rgba(100, 100, 100, 255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0.5, x2:0, y2:0, stop:0 rgba(255, 244, 228, 255), stop:1 rgba(255, 255, 255, 255));
}

QComboBox{
	font: 30pt;
	border:1px solid black;
	border-radius:30px;
	padding-left:40px;
	padding-bottom:3px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0.00568182, x2:0, y2:1, stop:0 rgba(255, 250, 233, 255), stop:1 rgba(255, 255, 255, 255));
}
QComboBox QAbstractItemView{
	outline: 0px solid black;
	border:1px solid grey;
	border-radius:10px;
	background-color: rgba(255, 255, 255, 0);
	selection-background-color :rgba(255, 255, 255, 0);
}
QComboBox QAbstractItemView:item{
	padding-left:40px;
	border:1px solid black;
	border-radius:20px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0.00568182, x2:0, y2:1, stop:0 rgba(255, 250, 243, 255), stop:1 rgba(255, 255, 255, 255));
}
QComboBox QAbstractItemView::item:selected{
	color:black;
	background-color: qlineargradient(spread:pad, x1:1, y1:0.00568182, x2:0, y2:1, stop:0 rgba(255, 250, 233, 255), stop:1 rgba(255, 255, 255, 255));
}

QLineEdit{
	font: 20pt;
	border: 1px solid black;
	border-radius: 20px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0.00568182, x2:0, y2:1, stop:0 rgba(255, 250, 233, 255), stop:1 rgba(255, 255, 255, 255));
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 250, 216);
	padding-left: 16px;
	padding-right: 16px;
}

QTextEdit{
	font: 20pt;
	border: 1px solid black;
	border-radius: 20px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0.00568182, x2:0, y2:1, stop:0 rgba(255, 250, 233, 255), stop:1 rgba(255, 255, 255, 255));
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 250, 216);
	padding-left: 16px;
	padding-right: 16px;
	padding-top: 10px;
	padding-bottom: 10px;
}

QSpinBox{
	font: 26pt;
	border:1px solid black;
	border-radius:10px;
 	background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:1, stop:0 rgba(255, 250, 233, 255), stop:1 rgba(255, 255, 255, 255));
	selection-color: rgb(0, 0, 0);
	selection-background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:1, stop:0 rgba(255, 250, 233, 255), stop:1 rgba(255, 255, 255, 255));
}

QScrollArea{
	border: none;
	background-color: transparent;
}

QTextBrowser{
	border: 1px solid black;
	background-color: rgba( 245, 245, 245, 200 );
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 250, 216);
}</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resource/resource.qrc">:/Image/background.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="Group_Friends" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>211</width>
     <height>481</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btn_makeFriends">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_myFriends">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>管理</string>
    </property>
   </widget>
   <widget class="Widget_FriendList" name="Group_FriendList" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#Group_FriendList{
	border: none;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>411</y>
      <width>191</width>
      <height>1</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::HLine</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Group_MyInfo" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border: 0px;
	border-radius: 30px;
}</string>
   </property>
   <widget class="QPushButton" name="btn_Info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_head">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>lb_name</zorder>
   <zorder>lb_head</zorder>
   <zorder>btn_Info</zorder>
  </widget>
  <widget class="QLineEdit" name="le_send">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>520</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>520</y>
     <width>91</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mail">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="Widget_Chat" name="Group_Chat" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>521</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget_Chat</class>
   <extends>QWidget</extends>
   <header>Form/Form_Main/widget_chat.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Widget_FriendList</class>
   <extends>QWidget</extends>
   <header>Form/Form_Main/widget_friendlist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
